docs/modules/ROOT/pages/explanations/improving-slos.adoc

```diff
@@ -0,0 +1,33 @@
+= Improving SLOs
+
+[abstract]
+This page provides high-level ideas on how the SLOs managed by this component can be improved over time.
+Providing detailed step-by-step instructions is a non-goal of this page.
+
+This component provides a number of SLOs, which may need to be improved or tuned for various reasons.
+Clusters may have special SLAs, which can't be met with the default SLO configuration.
+On clusters which use a non-standard configuration, some of the standard SLOs may generate non-actionable alerts.
+
+== Special SLA
+
+In case of special SLAs, the relevant SLO configurations should be updated to have an objective which satisfies the SLA.
+Ideally, the objective configured for the SLO should be higher than the objective documented in the SLA.
+Having a SLO which is stricter than the SLA reduces the number of SLA violations, because engineers are notified about issues after the SLO objective is violated.
+By notifying engineers early, the issue may get resolved before the SLA objective gets violated.
+
+See section "icon:wrench[] Tune" in the runbook of an SLO for detailed instructions on how to tune the configuration for that SLO.
+
+== Noisy SLO
+
+On clusters which have a non-standard configuration, some SLOs may generate a lot of alerts.
```

Comment:
One nitpick, this  sounds like we're pretty sure that the standard SLOs make sense and won't be noisy. As we basically pulled these out of our arse, that probably isn't the case.

Reply:
updated in latest version